### PR TITLE
Problems dashboard improvement NEW

### PR DESCRIPTION
**Pull request by marusak**
This is rework of #581

Mainly fixed:
  - build (editing specfile and Makefiles)
  - version_querying (which was not working and also was breaking tests)
  - Implemented tainted label
  - rebased to current code

This needs quite some testing, both functionality (I've done the basic, but testing with more diverged data is needed) and speed (will it be fast enough in the production?).

### CODE REVIEW

**Comment by mkutlak** [comment]
I found a quite a few issues.

**Comment by marusak** [comment]
> I found a quite a few issues

Great, can you explain them? :)

**Comment by mkutlak** [comment]
Some of the issues I thought were caused by this PR was just general UX.

 - Query in Advanced filters  ```Since version``` with value `.5` or (any number) causes crash (works on live FAF);  `,5` or `-5` works just fine.

```
InternalError: (psycopg2.InternalError) bad semver value '.5.0': expected number at .5.0
builds.semver <= '.5.0'
```

File: src/webfaf/templates/problems/problems_table.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
+{% macro problems_table(problems) %}
+    {% if problems %}
+      <noscript><p>WARNING!! This page is using JavaScript to filter and sort the table.</p></noscript>
+      <table class="table datatable table-striped table-bordered table-condensed">
+        <thead>
+          <tr>
+            <th>ID</th>
+            <th>Component</th>
+            <th>Count</th>
```
Comment (mkutlak) [inline src/webfaf/templates/problems/problems_table.html]: Does the change of order of the columns 
from 
`ID | Component | Crash function | State | External bugs | Probably fixed | Count `
to
`ID | Component | Count | Crash function | State | Bugs | Probably fixed `
help something? 

I would keep it as it is.

**Comment by marusak** [comment]
Updated and rebased

**Comment by mkutlak** [comment]
`Since/To version/release` = `a`

`InternalError: (psycopg2.InternalError) bad semver value '.0.0': expected number at .0.0`

**Comment by marusak** [comment]
Thank you @mkutlak for the reviews! I've updated both your issues.

**Comment by mkutlak** [comment]
All the previous issues have been fixed and I haven't found any other issue. 

Once the last one is fixed, we can merge this PR.

**Comment by marusak** [comment]
@mkutlak updated

**Comment by mkutlak** [comment]
LGTM. Thank you!

**Pull request merged**